packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -6,11 +6,162 @@
  * @prop --calcite-alert-width: Specifies the width of the component.
  */
 
+:host {
+  --calcite-alert-edge-distance: theme("spacing.8");
+  @apply block;
+}
+
+.container {
+  @apply bg-foreground-1
+  box-border
+  fixed
+  flex
+  flex-wrap
+  items-center
+  justify-center
+  mx-auto
+  my-0
+  opacity-0
+  pointer-events-none
+  shadow-2
+  text-start
+  w-full
+  z-toast;
+
+  container: responsive-container / inline-size;
+  border-radius: var(--calcite-border-radius);
+  border-block-start: 0 solid transparent;
+  border-inline: 1px solid var(--calcite-ui-border-3);
```

Comment:
I wonder if these border px values should come from a token?

Reply:
I stored them in a sass variable for now to DRY things up w/o adding a CSS prop.